scripts/generate_workers_cfg.sh

```diff
@@ -101,20 +117,50 @@ generate_workers () {
             if [ $(_jq '.gpu') = "true" ]; then
                 add_string_env_var NVIDIA_VISIBLE_DEVICES all
             fi
-            add_section networks
-            add_item mediacloudai_global
-            add_item workers
+
+            if [ $2 = "master" ]; then
+              add_section networks
+              add_item mediacloudai_global
+              add_item workers
+            fi
+
+            if [ $EFK_FLAG = true ]; then
+              add_section logging
+              depth 1
```

Comment:
Do you think it's clearer to write `depth 1; add_section_with_value driver "fluentd" ?`

Reply:
Definitely, I'll do the mod, it'll be clearer to everybody